Diverse Anpassungen

danspringer opened the pull request:
- Umstellung d. Pfade für heruntergeladene Schriften
- Generierung CSS pro Font
- Generierung gfonts.css für alle Fonts
- Abrufen der Schriftlizenzen
- Lizenzen mit in den Font-Ordner downloaden
- Preview für ausgewählte Fonts
- Hilfe-Seite / Fonts-Seite

danspringer committed:
Diverse Anpassungen

- Umstellung d. Pfade für heruntergeladene Schriften
- Generierung CSS pro Font
- Generierung gfonts.css für alle Fonts
- Abrufen der Schriftlizenzen
- Lizenzen mit in den Font-Ordner downloaden
- Preview für ausgewählte Fonts
- Hilfe-Seite / Fonts-Seite

AWqxKAWERbXo commented:
@danspringer hast du meinen PR #10 gesehen mit der Auslagerung der Textbausteine? Das ist jetzt ärgerlich, weil in deinem sind die wieder draußen.

danspringer committed:
Create .gitignore

danspringer commented:
Habs jetzt in einen neuen Commit übernommen. Danke! :-)

danspringer committed:
i18n übernommen, Ausgabe nach Download angepasst

- i18n-Übersetzungen von @alxndr-w übernommen und korrigiert https://github.com/AndiLeni/fonts/pull/10
- Darstellung nach Download der Fonts angepasst

danspringer committed:
Added Splashscreen

AWqxKAWERbXo commented:
@danspringer danke auch!
